Mykhailo_Honcharov/assignment1.py

```diff
@@ -0,0 +1,94 @@
+def main() -> None:
+    
+    # String Analysis
+
+    print("Hello!")
+    first_name = input("What is your first name?")
+    last_name = input("What is your last name?")
+
+    first_name = first_name.strip()
+    last_name = last_name.strip()
+
+    # 1. Print the lenght of the first name and last name.
+
+    print("First name length is", len(first_name))
+    print("Last name length is", len(last_name))
```

Comment:
```suggestion

    print("First name length is", len(first_name))
    print("Last name length is", len(last_name))
```
Code like this is self-explanatory, no need for comments.